CHANGELOG.rst

```diff
@@ -3,6 +3,50 @@ Changelog
 #########
 All notable changes to the kytos project will be documented in this file.
 
+UNRELEASED - Under development
+******************************
+Added
+=====
+
+Changed
+=======
+
+Deprecated
+==========
+
+Removed
+=======
+
+Fixed
+=====
+
+Security
+========
+
+[2018.1b1] - "dalva" beta1 - 2018-3-09
+**************************************
+Added
+=====
+- Added some new blueprints
+- Added Interface metadata informations
```

Comment:
- Now, we have few Entities inside the core (Switch, Interface and Link)
- Each Entity has metadata attribute (a dict)